layouts/shortcodes/appsec-getstarted-2-canary.md

```diff
@@ -1,6 +1,6 @@
    The library collects security data from your application and sends it to the Agent, which sends it to Datadog, where [out-of-the-box detection rules][202] flag attacker techniques and potential misconfigurations so you can take steps to remediate. 
    
-1.  **To see Application Security Monitoring threat detection in action, send known attack patterns to your application**. For example, trigger the [Security Scanner Detected][203] rule by running a file that contains the following curl script:
+3.  **To see Application Security Monitoring threat detection in action, send known attack patterns to your application**. For example, trigger the [Security Scanner Detected][203] rule by running a file that contains the following curl script:
```

Comment:
I don't know the best way to deal with this problem. Perhaps take out the number entirely? In the .NET docs, this step needs to be #4.

Reply:
@urseberry Right, but Markdown is smart, so it numbers it correctly in the built copy (if you check the preview, the .NET page does have it at number 4). I think the real problem was the indentation in the java.md file.